server_action_mass_edit/tests/test_mass_editing.py

```diff
@@ -0,0 +1,406 @@
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd. (support@serpentcs.com)
+# Copyright 2018 Aitor Bouzas <aitor.bouzas@adaptivecity.com)
+# Copyrithg 2020 Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from ast import literal_eval
+
+from odoo.exceptions import ValidationError
+from odoo.tests import Form, common, new_test_user
+
+from odoo.addons.base.models.ir_actions import IrActionsServer
+
+
+def fake_onchange_model_id(self):
+    result = {
+        "warning": {
+            "title": "This is a fake onchange",
+        },
+    }
+    return result
+
+
+@common.tagged("-at_install", "post_install")
+class TestMassEditing(common.TransactionCase):
+    def setUp(self):
```

Comment:
Would `setUpClass` work for the common test setup to avoid running the setup separately for each test case?